packages/expo-cli/src/commands/upload/IOSUploader.js

```diff
@@ -138,12 +170,14 @@ export default class IOSUploader extends BaseUploader {
 
   async _uploadToTheStore(platformData, buildPath) {
     const { fastlane } = this;
-    const { appleId, appleIdPassword, appName, language } = platformData;
+    const { appleId, appleIdPassword, appName, language, appleTeamId } = platformData;
     const { bundleIdentifier } = this._exp.ios;
 
-    const appleCreds = { appleId, appleIdPassword };
+    const appleCreds = { appleId, appleIdPassword, appleTeamId };
 
-    log('Ensuring the app exists on App Store Connect, this may take a while...');
+    log(
+      'Ensuring the app exists on App Store Connect, this may take a while... (even about half an hour)'
```

Comment:
Well... I don't know if we should say anything like that. I think it might have been only a case for you. We should do more tests before saying it may take half an hour because generally, it shouldn't.

Reply:
It happened at least twice that I waited that much... "a while" suggests couple minutes at best, so I was pretty surprised and canceled this a few times before waiting that much, which turned out as a successful upload.